graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -15,90 +15,6 @@
 namespace graphbolt {
 namespace sampling {
 
-using NodeTypeList = std::vector<std::string>;
-using EdgeTypeList =
-    std::vector<std::tuple<std::string, std::string, std::string>>;
-
-/**
- * @brief Structure representing heterogeneous information about a graph.
- *
- * Example usage:
```

Comment:
This example is still important, add it back to CSCSamplingGraph.

Esp. make it clear what's node_type_offset.